internal/turso/databases.go

```diff
@@ -26,8 +27,22 @@ type Database struct {
 
 type DatabasesClient client
 
-func (d *DatabasesClient) List() ([]Database, error) {
-	r, err := d.client.Get(d.URL(""), nil)
+func (d *DatabasesClient) List(options ...map[string]string) ([]Database, error) {
```

Comment:
Why not just `options map[string]string`?

Reply:
It required updating elsewhere we use `List` to pass `nil`. I was just trying to find a good way to make this optional.

Reply:
I see.

Reply:
Maybe add a comment to explain the trickery

Reply:
Update: Reverted to use `nil`